x-pack/plugins/security_solution/public/overview/containers/overview_risky_host_links/use_risky_host_links.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+
+import { useCallback, useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { useRiskyHostsComplete } from './use_risky_hosts';
+import { useAppToasts } from '../../../common/hooks/use_app_toasts';
+import { useKibana } from '../../../common/lib/kibana';
+import { inputsActions } from '../../../common/store/actions';
+import { LinkPanelListItem } from '../../components/link_panel';
+
+export const QUERY_ID = 'risky_hosts';
+const noop = () => {};
+
+export interface RiskyHost {
+  host: {
+    name: string;
+  };
+  risk_score: number;
+  risk: string;
+}
+
+/* eslint-disable-next-line  @typescript-eslint/no-explicit-any */
+const isRiskyHostHit = (item: any): item is RiskyHost =>
+  item &&
+  item.host &&
+  typeof item.host.name === 'string' &&
+  typeof item.risk_score === 'number' &&
+  typeof item.risk === 'string';
+
+const getListItemsFromHits = (items: RiskyHost[]): LinkPanelListItem[] => {
+  return items.map(({ host, risk_score: count, risk: copy }) => ({
+    title: host.name,
+    count,
+    copy,
+    path: '',
+  }));
+};
+
+export const useRiskyHostLinks = ({ to, from }: { to: string; from: string }) => {
+  const [isModuleEnabled, setIsModuleEnabled] = useState<boolean | undefined>(undefined);
```

Comment:
Could we put this into a hook as well, we might need this information in flyout.

Reply:
hey Angela, I took a look and it looks like we would have to make an additional request if we break it down. It would be possible however to reuse this hook and read the value `isModuleEnabled` and ignore the results. actually we could probably use conditional `size: 0` parameter. let me know what you think!